packages/terra-table/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -62,6 +63,11 @@ const propTypes = {
    */
   isActive: PropTypes.bool,
 
+  /**
+   * Boolean value indicating whether or not the header cell is focused.
+   */
+  isDisplayVisible: PropTypes.bool,
+
```

Comment:
Is the prop description correct? It seems like the value is used to show/hide the display text.

Reply:
Yeah, I made a mistake here.  Ugh!